projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+)
+
+func Execute() {
+	help := flag.Bool("h", false, "describes how to use go-ls command")
+	flag.Parse()
+
+	if *help {
+		flag.PrintDefaults()
+		fmt.Println("Usage: go-ls [directories]")
+		return
+	}
+	args := flag.Args()
+
+	if len(args) == 0 {
+		args = []string{"."}
+	}
+
+	for _, y := range args {
```

Comment:
Can you think of a more clear name for this variable than `y`? What does it represent?